src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -164,6 +164,9 @@ h1, h2, h3, h4 {
 .docblock > h6:first-child {
 	margin-top: 0;
 }
+.since, .version {
+	font-variant-numeric: oldstyle-nums;
```

Comment:
What's the reason behind this change?

Reply:
In the below screenshot of the new version, the three red lines are the same length. As you can see, the digits are the same height as the lowercase letters.

![377982871-80f450af-ccc8-46d4-82ea-d4ea428f9276](https://github.com/user-attachments/assets/acd1b60a-5ccf-45cd-9866-3cbc58646d18)

In the below screenshot of the old version, the red lines are also the same length. As you can see, the digits are taller than the lowercase letters.

![377982837-b5c5132d-1e5e-402e-ba19-1dea9e70ea6f](https://github.com/user-attachments/assets/7b253f31-64a1-45e5-90da-ef752a08f333)

The problem with the old version is that adding the digits makes the subheader appear closer to the main header, even if its CSS box doesn't move. This means that, in the old version, there is no "correct" spacing that will look right both with and without the digits.

Reply:
I don't feel that strongly about it. If you want to stick with the present-day format, it's okay.